### PR TITLE
Update testcases due to remove asking consent for null value claims

### PR DESCRIPTION
**Pull request by Wathsara**
#### Purpose
Please Note @subject

#### Note
PR to remove asking the consent for the null value claims from the users profile - https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1638

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevocationWithSessionTerminationTestCase.java [hunk 1]
```diff
@@ -348,4 +354,20 @@ private JSONObject responseObject(String endpoint, List<NameValuePair> postParam
...
         return json;
     }
+
+    /**
+     * Crete claim values array.
+     *
+     * @return Array of claim values.
+     */
+    protected ClaimValue[] getUserClaims() {
+        ClaimValue[] claimValues = new ClaimValue[1];
```
Comment (mevan-karu) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevocationWithSessionTerminationTestCase.java:364]:
```suggestion

        ClaimValue[] claimValues = new ClaimValue[1];
```

**Review by mevan-karu — APPROVED** [review]

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCSSOConsentTestCase.java [hunk 2]
```diff
@@ -75,6 +75,8 @@ public void testInit() throws Exception {
 
         initUser();
         createUser(user);
```
Comment (GANGANI) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCSSOConsentTestCase.java:77]: If this created user is not used in this test case. Shall we remove that along with this fix?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevocationWithSessionTerminationTestCase.java [hunk 3]
```diff
@@ -348,4 +354,20 @@ private JSONObject responseObject(String endpoint, List<NameValuePair> postParam
         }
         return json;
     }
+
+    /**
+     * Crete claim values array.
+     *
+     * @return Array of claim values.
+     */
+    protected ClaimValue[] getUserClaims() {
```
Comment (GANGANI) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevocationWithSessionTerminationTestCase.java:363]: Add new line

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCSSOConsentTestCase.java (hunk 2 above)
Comment (Wathsara) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCSSOConsentTestCase.java:77]: In the test case it asserts  `0_Email,1_First Name` for the consent. Therefore we need to login from the user created here in order to get the correct consent that tries to assert.

**Review by GANGANI — APPROVED** [review]

**Pull request merged**